mods/ra/rules/aircraft.yaml

```diff
@@ -255,6 +255,10 @@ HELI:
 		Description: Helicopter gunship armed\nwith multi-purpose missiles.\n  Strong vs Tanks, Aircraft\n  Weak vs Infantry
 	Health:
 		HP: 120
+	RejectsOrders:
+		Except: Attack
```

Comment:
Yes. `RejectsOrders` gets disabled with the airborne upgrade, hence it doesn't reject orders when airborne.

You should only be able to issue attack orders while the heli is on the ground, not any move or other orders.